x-pack/plugins/lens/server/routes/existing_fields.ts

```diff
@@ -53,8 +53,12 @@ export async function existingFieldsRoute(setup: CoreSetup<PluginStartContract>,
       },
     },
     async (context, req, res) => {
-      const [{ savedObjects, elasticsearch }, { data }] = await setup.getStartServices();
+      const [{ savedObjects, elasticsearch, uiSettings }, { data }] =
```

Comment:
This looks a lot like https://github.com/elastic/kibana/pull/118555 . 🤔

Reply:
sorry my mistake didn't realize that i am on master and merged the wrong branch...